<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="addTrack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>add track</string>
    </property>
   </widget>
   <widget class="QTableView" name="playlistview">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>111</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>901</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>addition</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="but_prev">
       <property name="text">
        <string>prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_rewind">
       <property name="text">
        <string>rewind</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_play">
       <property name="text">
        <string>play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_stop">
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_forward">
       <property name="text">
        <string>fast forward</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="but_next">
       <property name="text">
        <string>next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="volumeSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>1</y>
      <width>541</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentTrack">
       <property name="text">
        <string>text</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="currentTrackTime">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderTrackDuration">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="trackDuration">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="RadioROKS">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>radio_roks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HitFM">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>hitFM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RadioJazz">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>radioJazz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="KissFM">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>130</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>KissFM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Play">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>230</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionprev">
   <property name="text">
    <string>prev</string>
   </property>
  </action>
  <action name="actionplay">
   <property name="text">
    <string>play</string>
   </property>
  </action>
  <action name="actionpause">
   <property name="text">
    <string>pause</string>
   </property>
  </action>
  <action name="actionnext">
   <property name="text">
    <string>next</string>
   </property>
  </action>
  <action name="actionstop">
   <property name="text">
    <string>stop</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
